oci-library/oci-hol/java-management-lifecycle-management/introduction/introduction.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+## About Java Runtime Lifecycle Management
+
+Lifecycle Management (LCM) is a part of JMS's reporting and management infrastructure. JMS enables users to observe and manage the lifecycles of their Java SE runtimes (on-premises or in the Cloud) by performing LCM operations such as Removing Java Runtimes.
+
+As a customer, you can:
+  * View and identify the Java Runtimes.
+  * Remove unwanted Java Runtimes.
+  * View and understand various status and logs related to LCM work requests.
+
+## About this Workshop
+
+Java Management Service (JMS) is a reporting and management infrastructure integrated with Oracle Cloud Infrastructure Platform services to observe and manage your use of Java SE (on-premises or in the Cloud). Java Runtime can be viewed and identified in fleet after successful JMS setup on Host. Further, these Java Runtimes can be managed by performing various Lifecycle Management (LCM) operations.
```

Comment:
Not sure what is the casing to be followed, I think no need to capitalise "Host" ? Acronyms and Java Runtimes makes sense for me to capitalise